include/class.ticket.php

```diff
@@ -1079,7 +1079,7 @@ function sendAccessLink($user) {
             'body' => $content->getLocalBody($lang),
         ), $vars);
 
-        $email->send($user, Format::striptags($msg['subj']),
+        $email->send($user->getEmail(), Format::striptags($msg['subj']),
```

Comment:
Any explanation for the change here? The type of user is important when creating message-id in `class.mailer.php`

Reply:
`$user` is always an object and `send()` needs the email address instead otherwise you run into the error `substr() expects parameter 1 to be string, object given`.

Reply:
I don't believe that's correct statement - see send function in class.mailer.php

Reply:
Okay, I see what you mean. `send()` is supposed to return the email address. I will have to look deeper into why that error occurs.